website/docs/docs/build/metrics.md

```diff
@@ -6,6 +6,10 @@ keywords:
   - dbt metrics layer
 ---
 
+:::info Coming soon
+The dbt Semantic Layer is undergoing some sophisticated changes, enabling more complex metric definitions, efficient querying, and changes to the [dbt_metrics package](https://docs.getdbt.com/blog/deprecating-dbt-metrics). For more info, check out the [The dbt Semantic Layer: what's next?](https://www.getdbt.com/blog/dbt-semantic-layer-whats-next/) blog.
```

Comment:
Hmm. I think we need more stark language here because we'll be deprecating the dbt_metrics package.

Reply:
ok gotcha! i went too conservative! how about:

> The dbt Semantic Layer is undergoing some sophisticated changes, enabling more complex metric definitions and efficient querying. As part of these changes, the [dbt_metrics package](https://docs.getdbt.com/blog/deprecating-dbt-metrics) will be deprecated. For more info, check out the [The dbt Semantic Layer: what's next?](https://www.getdbt.com/blog/dbt-semantic-layer-whats-next/) blog

Reply:
> The dbt Semantic Layer is undergoing some sophisticated changes, enabling more complex metric definitions and efficient querying. As part of these changes, the dbt_metrics package will be deprecated and replaced with MetricFlow. For more info ....

How about that?

Reply:
nicee! sounds great thanks @callum-mcdata !